README.md

```diff
@@ -26,30 +26,86 @@ cd ./yul-dc-management
 touch .secrets
 ```
 
-## If this is your first time working in this repo or the Dockerfile has been updated you will need to pull your services
+## Install
+
+Clone the yul-dc-camerata repo and install the gem.
 
 ```bash
-  docker-compose pull web
+git clone git@github.com:yalelibrary/yul-dc-camerata.git
+cd yul-dc-camerata
+bundle install
+rake install
 ```
 
-## Starting the app
+## Updates
 
-- Start the web service
+You can get the latest version at any point by updating the code and reinstalling
+
+```bash
+cd yul-dc-camerata
+git pull origin master
+bundle install
+rake install
+```
+
+## General Use
+
+Once camerata is installed on your system, interactions happen through the 
+camerata command-line tool or through its alias `cam`.  The camerata tool can be 
+used to bring the development stack up and down locally, interact with the 
+docker containers, deploy, run the smoke tests and otherwise do development 
+tasks common to the various applications in the yul-dc application stack.
+
+All buildin commands can be listed with `cam help` and individual usage 
+information is available with `cam help COMMAND`.  Please note that deployment 
+commands (found in the `./bin` directory) are pass through and are therefor not 
+listed by the help command.  See th usage for those below. 
+
+To start the application stack, run `cam up` in the management directory. This starts all of the applications as they are 
+all dependencies of yul-management. Camerata is smart. If you start `cam up` from 
+a management code check out it will mount that code for local development 
+(changes to the outside code will affect the inside container). If you start the 
+`cam up` from the management application you will get the management code mounted
+for local development and the management code will run as it is in the downloaded 
+image. You can also start the two applications both mounted for development by 
+starting the management application with `--without blacklight` and the 
+blacklight application `--without solr --withouth db` each from their respective 
+code checkouts.
```

Comment:
this paragraph says "start the management app" twice. The first set (sentence 2) should probably say blacklight.